src/services/teams/hooks/index.js

```diff
@@ -169,6 +170,30 @@ const existId = (hook) => {
 	return hook;
 };
 
+/**
+ * checks if user has been deleted and assigning all folders to team owner
+ * @beforeHook
+ * @param {Object::hook} hook
+ * @return {Promise::hook}
+ */
+const manageFolders = (hook) => {
```

Comment:
You can add here async await and don't use then, cause await will already ensure that the result is waited for